### DISCUSSION TITLE
Workers stuck on low gear

### DISCUSSION THREAD

**Discussion by Erabior** [Courseplay/Courseplay_FS22#1697] [Q&A]
I set the maximum speeds for every single one of my vehicles practically to the max. Regardless, my seeder only goes 6 kph. If I start it going myself and put it on a higher gear then it will go faster but the second it turns, it stays on A4 instead of going to B4 or C4 to go faster. The tracker has the power to go faster, I can work it at 18 kph, it is a modded tractor and sowing machine. the tractor is certainly not the issue as my spraying course works just fine and goes along at max speed. Trying to see if this is something I have wrong or if it is a genuine bug. currently using what I believe to be  Version 7.0.1.20 on FS 22 PC

**Comment by Tensuko** [comment]
Thats a problem of the gear system of Giants + Helper.
We do not control the gears.
When you shift it manually it is nothing we can do to control it as the Giants helper doesn't do it.

Also when it is modded, there might be some Issue in the Mod.

**Reply by Erabior** [reply]
So I did notice that the worker was shifting gears during turns despite it refusing to do so while working. Also courseplay has no issue with the that same tractor while using the vanilla sprayer. I'm not sure if that provides any additional info.

**Reply by Tensuko** [reply]
A sowing machine is not a sprayer and take more PS to be dragged arround... this is not a CP Problem!

**Reply by Erabior** [reply]
I will say again, if I work the seeder myself my tractor has no difficulty bringing it up to speed. It is not a lack of power. The AI refuses to take it off first gear.

**Reply by Erabior** [reply]
something else I am observing is that the worker will sometimes change to higher gears but mostly gets stuck on lower gears
